web/Migrations/20220718092020_HasRequestedAdmin.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BricksAndHearts.Migrations
+{
+    public partial class HasRequestedAdmin : Migration
```

Comment:
Possibly a bit pedantic, but I generally like migrations to describe what they do, so would name this something like `AddHasRequestedAdminToUser`

Reply:
Will keep this in mind for future. Is it possible to rename this migration now or is it too late?

Reply:
You can undo a migration, delete it, and recreate it with a different name. Something like
```
// Update your database to the latest good migration
dotnet ef database update <name of previous migration>

// Then remove the bad one (the last one)
dotnet ef migrations remove

// Then create the new one
dotnet ef migrations add <new migration name>

// and update
dotnet ef database update
```

You might want to do some googling and check out these commands yourself but I think this should do the trick :)